docs/notebooks/plot_slopes.ipynb

```diff
@@ -0,0 +1,2503 @@
+{
```

Comment:
<p>Maybe the first part here is redundant with the previous paragraph?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/701/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>Yeah, I agree. Deleted and re-worded to "The code block above is equivalent to taking the mean of the <code>estimate</code> and uncertainty columns. For example:"</p><p> </p><p> </p>